assemblyline_v4_service/common/base.py

```diff
@@ -55,7 +55,7 @@ def _handle_execute_failure(self, exception, stack_info) -> None:
     def _success(self) -> None:
         self._task.success()
 
-    def execute(self, request: ServiceRequest) -> Dict[str, Any]:
+    def execute(self, request: ServiceRequest) -> None:
```

Comment:
I think removing the return type from the method signature altogether is a better approach, unless I'm mistaken and the `execute` method actually returns `None` every time.

Reply:
In python if there is no return statement None is automatically returned at the end of the function, same thing happens with a blank return statement. I think most type checkers can infer no return type but "-> None" is the way to be explicit.

Reply:
Sounds good to me!